lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1195,6 +1195,8 @@ fc_extras
     UD_UNITS_LON = ['degrees_east', 'degree_east', 'degree_e', 'degrees_e',
                     'degreee', 'degreese', 'degrees', 'degrees east',
                     'degree east', 'degree e', 'degrees e']
+    UNKNOWN_UNIT_STRING = str(cf_units.as_unit(None))
```

Comment:
A tiny point, but I think it would be better to be definite, and use `'unknown'` instead of `None`.

Reply:
Additional:
I think, from the code here, we don't actually _need_ these to match some canonical form, or even be Unit instances, as they are only used as defaults and assigned values.
In that case, you can simply have strings, with your own choice of form, and don't need to wrap them in `as_unit` at all.

Also, looking again at cf_units, another simple alternative is to use the "unit symbol"s, 
which is perhaps clearer and more unambiguous.
With those, you can just have `"?"` for unknown, and `"-"` for no-unit,  
which at least avoids any uppercase / lowercase uncertainties.